src/tvheadend/entity/Event.h

```diff
@@ -157,6 +169,9 @@ class Event : public Entity
   std::string m_cast;
   std::string m_categories;
   std::string m_aired;
+  std::string m_rating_label;  // Label like 'PG' or 'FSK 12'
+  std::string m_rating_icon;   // Path to graphic for the above label.
+  std::string m_rating_source; // Parental rating source.
```

Comment:
Could you use camelCase here as well to match the other fields?

Reply:
> Could you use camelCase here as well to match the other fields?

Done.